src/mitol/openedx/utils/__init__.py

```diff
@@ -0,0 +1 @@
+from .courses import *
```

Comment:
This should be an absolute import instead.

Reply:
Thanks for pointing it out.
I'll change it